clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -138,7 +138,7 @@ public DeleteTopicsResult deleteTopics(Collection<String> topics) {
      *
      * This operation is not transactional so it may succeed for some topics while fail for others.
      *
-     * It may take several seconds after AdminClient#deleteTopics returns
+     * It may take several seconds after the {@code DeleteTopicsResult} returns
```

Comment:
Isn't the same issue present in the other topic management methods?

Reply:
Fair point. But the others don't mention anything like this (except `createTopics`). Thus updating `createTopics` seems to make sense (to at least avoid miss leading phrases).

I would leave all others as-is.

Reply:
I agree that the same issue is present in many other admin methods.  However, `createTopics` and `deleteTopics` tend to be the ones that surprise users.  Users do things like create a topic and then expect to be able to delete it immediately, in a test.